posts/2020-09-25-signed-certificate-with-acme-200010.adoc

```diff
@@ -0,0 +1,135 @@
+blog_description: With Open Liberty 20.0.0.10 you can now install a signed certificate with ACME Certificate Authority with acmeCA-2.0 plus significant bug fixes
+---
+= Install a signed certificate with Automatic Certificate Management Environment Support 2.0 in Open Liberty 20.0.0.10
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+// tag::intro[]
+
+With Open Liberty 20.0.0.10 you can now install a certificate signed by a Certificate Authority (CA) using the Automatic Certificate Management Environment (ACME) protocol for improved testing or user experience. Also in this release there are a number of significant bug fixes.
+
+In link:{url-about}[Open Liberty] 20.0.0.10:
+
+* <<acme, Install a signed certificate with Automatic Certificate Management Environment Support 2.0 (acmeCA-2.0)>>
+* <<bugs, Significant bugs fixed in this release >>
+
+View the list of fixed bugs in link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A200010+label%3A%22release+bug%22+[20.0.0.10].
+// end::intro[]
+// tag::run[]
+[#run]
+
+== Run your apps using 20.0.0.10
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>20.0.0.10</version>
+    <type>zip</type>
+</dependency>
+----
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[20.0.0.10,)'
+}
+----
+Or if you're using Docker:
+[source]
+----
+FROM open-liberty
+----
+//end::run[]
+Or take a look at our link:{url-prefix}/downloads/[Downloads page].
+[link=https://stackoverflow.com/tags/open-liberty]
+image::img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]
+//tag::features[]
+[#acme]
+=== Install a signed certificate with Automatic Certificate Management Environment Support 2.0 (acmeCA-2.0)
+
+For default Transport Security (SSL/TLS) support, Open Liberty provides a self-signed certificate. With the With the Automatic Certificate Management Environment (ACME) Support 2.0 feature, a Certificate Authority (CA) signed certificate can be installed, providing a trusted certificate for an improved testing or user experience. The self-signed certificate from Open Liberty allows you to enable transport security right away, but most browsers will mark the certificate as insecure and provide a warning or error to the user when accessing a website with a self-signed certificate. Although a signed certificate solves this problem, it can be costly and may not be available during development and testing. Getting a CA signed certificate (for example, Lets Encrypt), provides a middle ground of a trusted certificate.
```

Comment:
Please update "Although a signed certificate solves..." to "Although a CA signed certificate solves..."

Reply:
Done